### ISSUE TITLE
Plugin renders empty padding at the bottom (Month View)

### ISSUE THREAD

**Issue by asiryk** [pyrochlore/obsidian-tracker#392]
## Expected Behavior

The rendered month view should fit where the calendar ends, and there should not be extra empty space.

## Current Behavior
The plugin renders an empty space below the calendar.

## Steps to Reproduce (for bugs)
plugin version: 1.14.0
obsidian version: 1.7.5
macOS: 15.0.1

I'm using the following query:

```tracker
searchType: tag
searchTarget: journal, daily
folder: Journal
datasetName: Daily Notes Streak, ''
dateFormat: YYYY-MM-DD-ddd
month:
    startWeekOn: Mon
```

## Context
It is visually annoying.

<img width="755" alt="Screenshot 2024-11-11 at 15 20 01" src="https://github.com/user-attachments/assets/61229514-d6f2-4c3a-b69b-3634851f61ec">

**Comment by AnasImr** [comment]
yeah that is weird, although this is a great plugin, the rendering on it is very janky tbh 💀, it could probably be temporarily fixed by a css wizard by trimming that excess height with a negative margin
